version_control/katalyst_by_codurance/css/elements/_buttons.css

```diff
@@ -63,3 +80,31 @@ input[type="submit"]:disabled, {
   text-decoration: none;
   transition: none;
 }
+
+
+.arrow-down {
+  display: flex;
+  width: max-content;
+  align-items: center;
+}
+
+.arrow-down:after {
+  content: url("https://3042464.fs1.hubspotusercontent-na1.net/hubfs/3042464/KATALYST%20BY%20CODURANCE/Microsite/katalyst_arrow.svg");
```

Comment:
Just to be a little bit picky (haha), would it be easy tp change the arrow from an :after with an image being loaded and everything to just a line awesome icon?

Reply:
I´m keeping this svg arrow. The lineawesome one it's a bit unbalanced and way longer making the button higher. Closing this conversation.